sections/oxygen_pre_deployment.md

```diff
@@ -147,15 +147,15 @@ Setup for 100% and 0% calibration.
 
 Issues with Winkler method at low oxygen concentrations (below 1 $\mu$mol/kg, some researchers consider that values below 20 $\mu$mol/kg shouldn't be use for the sensor calibration): 
 - There are various issues with Winkler at low oxygen concentrations as Winkler always biases towards too high oxygen: the detection limit of the method is around 1 uM {cite}`Langdon2010`, the oxygen absorbed in the plastic of the Niskin bottles might be transferred to  the water sampled (reference needed).
-- Within the core of the Peruvian oxygen minimum zones oxygen concentrations at nmol levels are present
+- For example, within the core of the Peruvian oxygen minimum zones oxygen concentrations at nmol levels are present
 other approaches for 0 % in-situ calibration {cite}`Revsbech2009` and {cite}`Thomsen2016`.
-- STOX sensor, new lox-oxygen sensing foils (0-10% saturation) from Aanderaa.
+- STOX sensor, new low-oxygen sensing foils (0-10% saturation) from Aanderaa.
 
 #### 100 / 0 % saturation protocol
 
-This method is necessary to check drift over time as the foil wears out. It’s recommended to do it before the deployment and after recovery. NOTE: Sensor foil must be wet during all processes.   
+This method is necessary to check the drift of oxygen measurements over time as the foil wears out. It’s recommended to do it before the deployment and after recovery. NOTE: Sensor foil must be wet during all protocol steps.   
 Some information to read before we proceed with the calibration:
-1. For a 100 % bubbled bath, connect an aquarium pump  to a tube which has been fitted with a porous stone (bubble dispenser) at the end. This will create small air bubbles that are sufficient to equilibrate the water rapidly. 
+1. For a 100 % bubbled bath, connect an aquarium pump to the tube which at the end you have fitted with a porous stone (bubble dispenser). This will create small air bubbles that are sufficient to equilibrate the water rapidly. 
```

Comment:
I prefer to not talk to the user in "you" but keep in passive as it is.

Reply:
You are right @soerenthomsen